crates/astria-cli/src/sequencer/sudo/fee_change.rs

```diff
@@ -0,0 +1,522 @@
+use astria_core::protocol::{
+    fees::v1::{
+        BridgeLockFeeComponents,
+        BridgeSudoChangeFeeComponents,
+        BridgeUnlockFeeComponents,
+        FeeAssetChangeFeeComponents,
+        FeeChangeFeeComponents,
+        IbcRelayFeeComponents,
+        IbcRelayerChangeFeeComponents,
+        IbcSudoChangeFeeComponents,
+        Ics20WithdrawalFeeComponents,
+        InitBridgeAccountFeeComponents,
+        RollupDataSubmissionFeeComponents,
+        SudoAddressChangeFeeComponents,
+        TransferFeeComponents,
+        ValidatorUpdateFeeComponents,
+    },
+    transaction::v1::{
+        action::FeeChange,
+        Action,
+    },
+};
+use clap::Subcommand;
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+
+use crate::utils::submit_transaction;
+
+#[derive(Debug, clap::Args)]
+pub(super) struct Command {
+    #[command(subcommand)]
+    command: SubCommand,
+}
+
+impl Command {
+    pub(super) async fn run(self) -> eyre::Result<()> {
+        match self.command {
+            SubCommand::TransferFee(transfer) => transfer.run().await,
+            SubCommand::InitBridgeFee(bridge_init) => bridge_init.run().await,
+            SubCommand::RollupDataSubmissionFee(rollup_data) => rollup_data.run().await,
+            SubCommand::BridgeLockFee(bridge_lock) => bridge_lock.run().await,
+            SubCommand::BridgeUnlockFee(bridge_unlock) => bridge_unlock.run().await,
+            SubCommand::BridgeSudoChangeFee(bridge_sudo_change) => bridge_sudo_change.run().await,
+            SubCommand::Ics20WithdrawalFee(ics20_withdrawal) => ics20_withdrawal.run().await,
+            SubCommand::IbcRelaeyFee(ibc_relay) => ibc_relay.run().await,
+            SubCommand::IbcRelayerChangeFee(ibc_relayer_change) => ibc_relayer_change.run().await,
+            SubCommand::IbcSudoChangeFee(ics_sudo_change) => ics_sudo_change.run().await,
+            SubCommand::FeeAssetChangeFee(fee_asset_change) => fee_asset_change.run().await,
+            SubCommand::FeeChangeFee(fee_change) => fee_change.run().await,
+            SubCommand::SudoAddressChangeFee(sudo_address_change) => {
+                sudo_address_change.run().await
+            }
+            SubCommand::ValidatorUpdateFee(validator_update) => validator_update.run().await,
+        }
+    }
+}
+
+#[allow(clippy::enum_variant_names)]
+#[derive(Debug, Subcommand)]
+enum SubCommand {
+    /// Chnage Transfer Fee
+    TransferFee(Transfer),
+    /// Change Init Bridge Account Fee
+    InitBridgeFee(BridgeInit),
+    /// Change Sequence Fee
```

Comment:
Would it be better to call this `Rollup Data Submission Fee`? I'm not opinionated either way, but maybe worth considering.